ci/kind/kind-setup.sh

```diff
@@ -287,7 +289,16 @@ function configure_vlan_subnets {
     done
 
     docker_run_with_host_net iptables -t filter -A FORWARD -i $bridge_interface -o $vlan_interface -j ACCEPT
-    docker_run_with_host_net iptables -t filter -A FORWARD -o $bridge_interface -i $vlan_interface -j ACCEPT
+    docker_run_with_host_net iptables -t filter -A FORWARD -i $vlan_interface -o $bridge_interface -j ACCEPT
+    docker_run_with_host_net iptables -t filter -A FORWARD -i $vlan_interface -o $vlan_interface -j ACCEPT
```

Comment:
out-of-curiosity, when do we have that type of hair-pinned traffic?

Reply:
I think it's for this case `TestAntreaIPAMService/testAntreaIPAMVLAN11PodToAntreaIPAMVLAN11ClusterIPv4/Non-HostNetwork_Endpoints/Connect_to_Service_ClusterIP_from_Pod`, not all vlan interfaces need this rule, There should be only vlan 11 needs it, but we added here in case there will be new tests required the same rule in the future.

Reply:
Interesting, thanks. I didn't think of Service traffic.